### PR TITLE
Solar fix

### PR DESCRIPTION
**Pull request by garrulousTeamotei**
## About The Pull Request
Fixes Solars. Previously (Since about 2017, it seems.) it was impossible to create a solar array due to the inability to add glass to the solar assembly. It was impossible to disassemble a solar array without breaking one (If an admin ever spawned on during that time). if you wrenched down a solar array it said you unwrenched it, and vice versa. Adding a solar array to a bag makes it impossible to remove, not to mention you could wrench and unwrench it in a bag. I have no idea how to fix that last one, so I just increased the size of the assembly to not fit in bags. Still can fit a good 10+ in lockers, and more in crates. Looks big enough to not feasibly fit in a bag, anyways. The sprite for the Solar Control Console was fixed from a red "X" To the default computer console sprite

## Why It's Good For The Game

Fixing a power generator system is good, right? We loose a singularity engine from the cargo changes and get a functioning solar array to fill it's place.. 

## Changelog
:cl:

tweak: Solar Assembly changed from Bulky to Huge
fix: Solar Arrays can be made
fix: Solar Arrays can be disassembled
fix: Wrenching Solar Assemblies gives the correct text.
fix: Solar Control Console given a working default sprite

/:cl:

### CODE REVIEW

**Comment by ValoTheValo** [comment]
Based PR

**Comment by garrulousTeamotei** [comment]
Literally took me like half an hour, and I'm barely a coder.

**Comment by ValoTheValo** [comment]
> Literally took me like half an hour, and I'm barely a coder.

still extraordinarily based

**Comment by garrulousTeamotei** [comment]
Uh. Looks like my Sword of Truth PR piggybacked on this PR. Even though it was already merged . . . Is that a problem, or does it need to be fixed? No idea how to fix it though.

**Comment by ValoTheValo** [comment]
relevant
image

**Comment by garrulousTeamotei** [comment]
Ah yes. As the ship moves fast through space, it will get very windy. So windy that big flat boards of silicon will be blown off. Very unrealistic that solars would work on the Eris. Better add some code so they break if it's on Z-levels 1-5 and the Eris moves.
Gotta thank Kegdo for being so observant and well learned.

**Review by SirRichardFrancis — COMMENTED** [review]
I'd rather see solars yeeted out of the build, to be honest.
For the console you just set `icon` and `icon_state`, if original sprites lost in time - take generic sprite from other console.

File: code/modules/power/solar.dm [hunk 1]
```diff
@@ -294,6 +294,8 @@
 	var/nexttime = 0		// time for a panel to rotate of 1� in manual tracking
 	var/obj/machinery/power/tracker/connected_tracker = null
 	var/list/connected_panels = list()
+	var/icon_keyboard ="generic_key"
+	var/icon_screen ="generic"
```
Comment (TheShown911) [inline code/modules/power/solar.dm:298]:
```suggestion
	var/icon_keyboard = "generic_key"
	var/icon_screen = "generic"
```

File: code/modules/power/solar.dm [hunk 2]
```diff
@@ -350,23 +352,29 @@
...
-	if(cdir > -1)
-		add_overlays(image('icons/obj/computer.dmi', "solcon-o", FLY_LAYER, angle2dir(cdir)))
-	return
+	else
+		set_light(light_range_on, light_power_on)
+
+	if(stat & BROKEN)
+		overlays += image(icon,"[icon_state]_broken")
+	else
+		overlays += image(icon,icon_screen)
```
Comment (TheShown911) [inline code/modules/power/solar.dm:373]:
```suggestion
		overlays += image(icon, "[icon_state]_broken")
	else
		overlays += image(icon, icon_screen)
```

**Review by TheShown911 — APPROVED** [review]
If we have solars, they might as well work

**Comment by TheShown911** [comment]
I think resolving the conflict by rebasing to master should fix sword of truth stuff

**Comment by hyperioo** [comment]
Still conflicting

**Comment by garrulousTeamotei** [comment]
Yea, rebasing isn't working out. I'm just gonna scrap this and try again.

**Pull request closed without merging**